packages/actions/src/Concerns/CanImportRecords.php

```diff
@@ -397,7 +397,8 @@ public function getUploadedFileStream(TemporaryUploadedFile $file)
             $s3Adapter = Storage::disk($fileDisk)->getAdapter();
 
             invade($s3Adapter)->client->registerStreamWrapper(); /** @phpstan-ignore-line */
-            $fileS3Path = 's3://' . config("filesystems.disks.{$fileDisk}.bucket") . '/' . $filePath;
+            
+            $fileS3Path = str_replace('\\','/','s3://' . config("filesystems.disks.{$fileDisk}.bucket") . '/' . $filePath);
```

Comment:
I think this is slightly more readable, what do you think?

```suggestion
            $fileS3Path = (string) str('s3://' . config("filesystems.disks.{$fileDisk}.bucket") . '/' . $filePath)->replace('\\', '/');
```

Reply:
Yes sir! I just went with str_replace for compatibility.